rfcs/0094-use-matrix-for-official-chat.md

```diff
@@ -0,0 +1,62 @@
+---
+feature: use-matrix-for-official-chat
+start-date: 2020-05-19
+author: Ryan Mulligan
+shepherd-team:
+shepherd-leader:
+---
+
+# Summary
+[summary]: #summary
+
+Switch from IRC ([Freenode](https://freenode.net/)) to
+[Matrix](https://matrix.org/)
+([#community:nixos.org][1]) as
+our official community chat.
+
+# Motivation
+[motivation]: #motivation
+
+The current official community chat host, Freenode, has lost its
+leadership and its future is uncertain. Many Free Software communities
+are abandoning it, so its network effects may be negative instead of
+positive.
+
+There are currently 1796 members of the Unofficial NixOS Discord
+channel (244 weekly visitors, 95 weekly communicators). Some of them
+might be convinced by the more modern features available on Matrix to
+move their conversations there.
+
+# Detailed design
+[design]: #detailed-design
+
+Update the [NixOS.org community
+page](https://nixos.org/community/index.html) to point to our Matrix
+Space ([#community:nixos.org][1]) that has already been created which you
+can join by visting the [invite
+link][1].
```

Comment:
Do we have a list of all the #nix- and #nixos- channels somewhere? What will they be mapping to? If we do that we should probably update all the channel topics so people know where to go.

Reply:
That's the job of the space. Channel lists in topics are only relevant until spaces have a wider client support.